index.html

```diff
@@ -334,7 +380,43 @@ <h2 class="contact-title">Contact me</h2>
         </div>
       </section>
     </section>
-
+    <div class="popup-window" id="popup">
+      <section class="popup">
+        <div class="popup-header">
+          <h2 class="popup-project-title" id="popup-project-title"></h2>
+          <span class="close">&times;</span>
+        </div>
+        <div class="popup-project-info" id="popup-project-info"></div>
+        <div id="popup-project-image-holder">
```

Comment:
- [ ] Do you think maybe we could improve on the styles from the header, as they are right now they're still very different from the Figma Template. Remember that you can copy and paste from Figma colors, size, margins, etc.

For reference, yours:
![image](https://user-images.githubusercontent.com/79114233/141317961-1aef2fa3-dd83-4752-b0cb-b3af00173dab.png)

 Figma: 
![image](https://user-images.githubusercontent.com/79114233/141317994-6147c890-3516-478e-a781-aa2b353d5f7b.png)